### PR TITLE
[Documentation] Architecture - Symfony, Doctrine & Twig

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | yes
| New docs?    | yes
| BC breaks?      | no
| Related tickets | partially #5275 
| License         | MIT

### CODE REVIEW

File: docs/book/architecture.rst [hunk 1]
```diff
@@ -11,6 +11,42 @@ All bundles share the same conventions for naming things and the way of data per
...
 For deeper understanding of how Doctrine works, please refer to the `excellent documentation on their official website <http://doctrine-orm.readthedocs.org/en/latest/>`_.
 
+Fullstack Symfony
+-----------------
+
+.. image:: ../_images/symfonyfs.png
+    :scale: 15%
+    :align: center
+
+**Sylius** is based on Symfony2, which is an Open Source PHP Web applications development framework. Using Symfony allows
```
Comment (pamil) [inline docs/book/architecture.rst]: "Open Source PHP Web applications development framework"

It's quite complicated explanation 🌿

File: docs/book/architecture.rst [hunk 2]
```diff
@@ -11,6 +11,42 @@ All bundles share the same conventions for naming things and the way of data per
...
+developers to work better and faster by providing them with certainty of developing an application that is fully compatible
+with the business rules, that is structured, maintainable and upgradable, but also it allows to save time by providing generic re-usable modules.
+
+To get to know more about Symfony - look `here <http://symfony.com/what-is-symfony>`_.
```
Comment (pamil) [inline docs/book/architecture.rst]: What about linking the whole phrase like:
```
`Learn more about Symfony <http://symfony.com/what-is-symfony>`_
```

File: docs/book/architecture.rst (hunk 1 above)
Comment (michalmarcinkowski) [inline docs/book/architecture.rst]: Maybe simply "The leading PHP framework to create web applications"?

Comment (pamil) [inline docs/book/architecture.rst]: @michalmarcinkowski do you mean https://laravel.com/?

**Comment by pjedrzejewski** [comment]
Thank you Magda! 👍 👍 👍

**Pull request merged**